src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -495,6 +537,11 @@ public AllocatedEndpoint(EndpointAnnotation endpoint, string address, int port,
         public override string ToString() { throw null; }
     }
 
+    public static partial class AspireStoreExtensions
+    {
+        public static IAspireStore CreateStore(this IDistributedApplicationBuilder builder) { throw null; }
```

Comment:
I don't think this should be an extension method off `IDistributedApplicationBuilder`. It means that all app devs will see it in their Intellisense when 99% of them don't need to use this class/API. Can it just be a "normal" static method?

Reply:
I don't think it should be in the extensions class either 😄

Reply:
@sebastienros ?